src/routes/UserSettings/EditProfileTab/__snapshots__/EditProfileTab.test.js.snap

```diff
@@ -6,7 +6,7 @@ Array [
     "Please enter the full url for your LinkedIn page.",
   ],
   Array [
-    "Invalid url. Please enter full url for your LinkedIn page.",
+    "Invalid url. Please enter the full url for your LinkedIn page.",
```

Comment:
I noticed an inconsistency in wording between the first and second prompt. (Second prompt is missing the word "the").
<img width="693" alt="Screen Shot 2021-03-24 at 8 46 45 PM" src="https://user-images.githubusercontent.com/38114845/112416000-0e93d900-8ce2-11eb-9f28-ace2eeea8cf7.png">

I just standardized the wording and updated the test snapshot!

Reply:
thanks!